<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>775</width>
    <height>516</height>
   </rect>
  </property>
  <property name="contextMenuPolicy">
   <enum>Qt::DefaultContextMenu</enum>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true">QGroupBox{
	font: 22px;
}</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>20</y>
      <width>251</width>
      <height>51</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QLabel{
	font:30px;
	color:#444444;
}</string>
    </property>
    <property name="text">
     <string>实验二：电子钢琴</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>100</y>
      <width>311</width>
      <height>191</height>
     </rect>
    </property>
    <property name="title">
     <string>功能一：自动播放</string>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>281</width>
       <height>41</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
	font-size: 15px;
	color: #666666;
}</string>
     </property>
     <property name="text">
      <string>每间隔0.5秒扬声器发出不同频率的声音，每次
升高100Hz，直到频率达到2000Hz。</string>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_1">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>150</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>点击播放</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>100</y>
       <width>111</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>初始频率设置：</string>
     </property>
    </widget>
    <widget class="QSpinBox" name="digital">
     <property name="geometry">
      <rect>
       <x>180</x>
       <y>100</y>
       <width>61</width>
       <height>22</height>
      </rect>
     </property>
     <property name="keyboardTracking">
      <bool>true</bool>
     </property>
     <property name="maximum">
      <number>1900</number>
     </property>
     <property name="singleStep">
      <number>100</number>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>250</x>
       <y>100</y>
       <width>41</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>/Hz</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_2">
    <property name="geometry">
     <rect>
      <x>390</x>
      <y>100</y>
      <width>351</width>
      <height>191</height>
     </rect>
    </property>
    <property name="title">
     <string>功能二：字符串播放</string>
    </property>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>80</y>
       <width>81</width>
       <height>16</height>
      </rect>
     </property>
     <property name="text">
      <string>选定节拍：</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_6">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>110</y>
       <width>72</width>
       <height>15</height>
      </rect>
     </property>
     <property name="text">
      <string>输入字串：</string>
     </property>
    </widget>
    <widget class="QLineEdit" name="stringLine">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>110</y>
       <width>231</width>
       <height>21</height>
      </rect>
     </property>
    </widget>
    <widget class="QPushButton" name="btn_2">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>150</y>
       <width>93</width>
       <height>28</height>
      </rect>
     </property>
     <property name="text">
      <string>点击播放</string>
     </property>
    </widget>
    <widget class="QLabel" name="label_7">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>281</width>
       <height>31</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">QLabel{
	font-size: 15px;
	color: #666666;
}</string>
     </property>
     <property name="text">
      <string>选定基准节拍，输入一串字符串（a到u），
可输出一段音乐。</string>
     </property>
    </widget>
    <widget class="QComboBox" name="beatOption">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>80</y>
       <width>101</width>
       <height>22</height>
      </rect>
     </property>
     <property name="editable">
      <bool>false</bool>
     </property>
     <property name="currentText">
      <string/>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox_3">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>330</y>
      <width>711</width>
      <height>131</height>
     </rect>
    </property>
    <property name="title">
     <string>控制台</string>
    </property>
    <widget class="QTextBrowser" name="res_console">
     <property name="geometry">
      <rect>
       <x>20</x>
       <y>30</y>
       <width>671</width>
       <height>91</height>
      </rect>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
